### ISSUE TITLE
Requesting TensorRT output vector format documentation

### ISSUE THREAD

**Issue by dschmidtOPT** [roboflow/rf-detr#264]
Hello - 

I have a (seemingly) working fine tuned RF-DETR Base model inference setup on an RTSP stream, and the model output vector is 1200 float values between 0-1.  Is there any TensorRT specific documentation for RF-DETR or predecessor models on how to parse the output?  Specifically - is there a way to determine how many class logits to expect with N class categories?  My assumption is that the number of class logits should be N + 1 for the null classification.

My specific example:
-model has 17 class categories
-assuming a transformer output structure of [cx, cy, w, h] + class_logits ( including 1 no - classification logit ), I would expect 22 values per classification, which does not map correctly.  
-20 values per classification, 24 values, or other factors of 1200 map, but its unclear which value to use.

Thank you for whatever information you are able to provide.

**Comment by Borda** [comment]
TensorRT export documentation was added in v1.5.0. Use `.export()` for ONNX output, then `trtexec` for TensorRT conversion.

See: https://github.com/roboflow/single_artifact_benchmarking for end-to-end examples including output parsing.

To upgrade: `pip install rfdetr>=1.6.1`

Closing as addressed. Feel free to reopen if more documentation is needed!
